Source/WebKit/Modules/Internal/module.modulemap

```diff
@@ -0,0 +1,4 @@
+module WebKit_Internal [system] {
```

Comment:
per https://clang.llvm.org/docs/Modules.html#module-maps:
> The system attribute specifies that the module is a system module. When a system module is rebuilt, all of the module’s headers will be considered system headers, which suppresses warnings. This is equivalent to placing #pragma GCC system_header in each of the module’s headers.

I would think we do _not_ want these semantics, since this module is never used by clients of WebKit.